asr-noise-augmentation.ipynb

```diff
@@ -0,0 +1,605 @@
+    "- Text-to-Speech synthesis (TTS)\n",
+    "- A collection of natural language processing (NLP) services, such as named entity recognition (NER), punctuation, and intent classification.\n",
+    "\n",
+    "In this tutorial, we will show how to augment your training data with background noise data for fine-tuning the Acoustic Model (Conformer-CTC) to improve accuracy on audio with background noise.  \n",
+    "To understand the basics of Riva ASR APIs, refer to [Getting started with Riva ASR in Python](https://github.com/nvidia-riva/tutorials/blob/stable/asr-python-basics.ipynb). <br>\n",
+    "\n",
+    "For more information about Riva, refer to the Riva [product page](https://developer.nvidia.com/riva) and [documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/overview.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data preprocessing\n",
+    "\n",
+    "For fine-tuning we need audio data with background noise. If you already have such data, then you can use it directly.  \n",
+    "In this tutorial, we will take the AN4 dataset and augment it with noise data from the Room Impulse Response and Noise Database from the [openslr database](https://www.openslr.org/28/).\n",
+    "\n",
+    "In this tutorial, we will be using NVIDIA NeMo for the data preprocessing step.\n",
+    "\n",
+    "#### NVIDIA NeMo Overview\n",
+    "\n",
+    "NVIDIA NeMo is a toolkit for building new state-of-the-art conversational AI models. NeMo has separate collections for Automatic Speech Recognition (ASR), Natural Language Processing (NLP), and Text-to-Speech (TTS) models. Each collection consists of prebuilt modules that include everything needed to train on your data. Every module can easily be customized, extended, and composed to create new conversational AI model architectures.\n",
+    "For more information about NeMo, refer to the [NeMo product page](https://developer.nvidia.com/nvidia-nemo) and [documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/intro.html). The open-source NeMo repository can be found [here](https://github.com/NVIDIA/NeMo).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Requirements and setup for data preprocessing:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Requirements:\n",
+    "\n",
+    "We will be using [NVIDIA NeMo](https://github.com/NVIDIA/NeMo) for this data preprocessing step - Easiest way to install and run NeMo is through NVIDIA's PyTorch docker container. If you are not already running this notebook in the NVIDIA PyTorch docker container, please follow instructions [here](./README.md#running-the-nvidia-riva-tutorial-how-to-improve-accuracy-on-specific-speech-patterns-by-fine-tuning-the-acoustic-model-citrinet-in-the-riva-asr-pipeline) to re-run this tutorial from the PyTorch docker container\n",
```

Comment:
```suggestion
    "We will be using [NVIDIA NeMo](https://github.com/NVIDIA/NeMo) for this data preprocessing step. The easiest way to install and run NeMo is through NVIDIA's PyTorch Docker container. If you are not already running this tutorial in the NVIDIA PyTorch Docker container, follow the instructions to do [here](./README.md#running-the-nvidia-riva-tutorial-how-to-improve-accuracy-on-specific-speech-patterns-by-fine-tuning-the-acoustic-model-citrinet-in-the-riva-asr-pipeline).\n",
```

Reply:
I rewrote this section. See https://github.com/nvidia-riva/tutorials/pull/120/commits/ddb2fd9b19215e8ec7b2d273e5d7525eee3e6b1b.